### ISSUE TITLE
Support multiple Ubuntu distros

### ISSUE THREAD

**Issue by wooksong** [nnstreamer/TAOS-CI#647]
Currently, the Ubuntu build checker in TAOS-CI works only for Xenial, which reaches EOL. Moreover, in some cases, it is possible to face different results that are come from differences between the distribution variations. 

I think ```pbuilder``` and ```pdebuild``` have already supported such features so that we could exploit them.

**Comment by myungjoo** [comment]
Yes.. and it'd be also appropriate to move on to 18.04 (and 20.04 optionally).
